modules/currency-l0/src/main/scala/org/tessellation/currency/l0/snapshot/programs/Rollback.scala

```diff
@@ -44,7 +44,7 @@ object Rollback {
     private val logger = Slf4jLogger.getLogger[F]
 
     def rollback: F[Unit] = for {
-      (globalSnapshot, globalSnapshotInfo) <- globalL0Service.pullLatestSnapshot
+      (globalSnapshot, globalSnapshotInfo) <- globalL0Service.pullLatestSnapshot(skipMajorityCheck = true)
```

Comment:
@TheMMaciek because those are private, i.e. not part of the interface. This solution is one approach but I'm open to changing the interface if everyone's ok with that.